src/pages/resultsView/plots/PlotsTab.tsx

```diff
@@ -2206,6 +2237,30 @@ export default class PlotsTab extends React.Component<IPlotsTabProps,{}> {
         return ( (this.vertSelection.dataType === AlterationTypeConstants.GENERIC_ASSAY || this.horzSelection.dataType === AlterationTypeConstants.GENERIC_ASSAY)
                   && this.vertTreatmentOptions.length === 0);
     }
+    
+    @computed get showUtilitiesMenu() {
+        return (this.plotDataExistsForTwoAxes || this.waterfallPlotIsShown) && (this.mutationDataCanBeShown || this.cnaDataCanBeShown);
+    }
+    
+    @computed get sharedPlotTitle():string {
+        if (this.showClinicalAttrsWithTreatments) {
+            const selection = this.horzSelection.dataType === AlterationTypeConstants.GENERIC_ASSAY ? this.horzSelection: this.vertSelection;
+            const treatmentName = selection.selectedTreatmentOption!.label;
+            const profileName = selection.selectedDataSourceOption!.label;
```

Comment:
For a clinical vs treatment profile scatter plot the title should reference both axes.  I may get very long, but just see and discuss when we see the result.

Reply:
The plot title becomes long indeed, and there are cases when it is overflowing outside of the svg area. Unfortunately, there is no text-wrap property in svg, so I'm still trying to figure out how to dynamically breaks the title to multiple lines based on the svg width.

A screenshot for reference:
![Screenshot from 2019-11-07 12-52-58](https://user-images.githubusercontent.com/31291004/68387188-0b846100-015e-11ea-93d7-c2c130663a5d.png)

Reply:
So here's what it looks like:
![Screenshot from 2019-11-07 13-36-27](https://user-images.githubusercontent.com/31291004/68389692-d844d080-0163-11ea-84ed-989ba79c669a.png)